### DISCUSSION TITLE
Unable to Update to Newest Version of Paru due to Incorrect libalpm Version

### DISCUSSION THREAD

**Discussion by electricsurge31** [Morganamilo/paru#400] [Q&A]
I have just tried to upgrade to paru v1.7.0-1 and I am getting this error:

```
error: failed to run custom build command for `alpm v2.0.0`

Caused by:
  process didn't exit successfully: `/var/tmp/pamac-build-esurge/paru/src/paru-1.7.0/target/release/build/alpm-e4058bce6d1c07dd/build-script-build` (exit code: 101)
  --- stderr
  thread 'main' panicked at 'this version of alpm.rs does not support libalpm v12.0.2 only v13.x.0 is supported', /home/esurge/.cargo/registry/src/github.com-1ecc6299db9ec823/alpm-2.0.0/build.rs:18:9
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
==> ERROR: A failure occurred in build().
    Aborting...
```

I'm really not sure what to do about this here as it seems i cannot just upgrade directly to a new version (I apologize if this seems trivial, I'm not well versed in the package management area).

I am using Manjaro, and the only thing i can come up with is that i may need to wait for them to release an update? Since they are normally not tracking mainline Arch directly?
If I am completely off here please let me know! Any help would be appreciated!

Thanks in advance!

**Comment by Morganamilo** [comment]
There's already a ton of issues on this: https://github.com/Morganamilo/paru/issues/398
